src/guides/v2.3/javascript-dev-guide/widgets/widget_gallery.md

```diff
@@ -356,6 +356,15 @@ Sliding direction of thumbnails in the fullscreen view.
 *  `vertical`
 *  `horizontal`
 
+```javascript
+	<var name="options">
```

Comment:
Please, change the nested "options" example to the real-world example, so we have 

```xml
<var name="gallery">
    <var name="navdir">horizontal</var>
    ...
</var>
```

Also, change the `javascript` code block type to `xml`.

Thank you!